builder/internals.go

```diff
@@ -489,8 +489,11 @@ func (b *Builder) processImageFrom(img *imagepkg.Image) error {
 		b.Config = img.Config
 	}
 
-	if len(b.Config.Env) == 0 {
-		b.Config.Env = append(b.Config.Env, "PATH="+daemon.DefaultPathEnv)
+	// No default path on Windows (set by HCS)
+	if runtime.GOOS != "windows" {
+		if len(b.Config.Env) == 0 {
+			b.Config.Env = append(b.Config.Env, "PATH="+daemon.DefaultPathEnv)
```

Comment:
The only place that variable is used is in processImageFrom in builder\internals.go. It was easier to keep compilation happy by keeping it declared and making it a no-op usage on Windows in internals.go. The alternate is remove the variable from container_windows and making the assignment in processImageFrom into a "func SetDefaultPath()" which would be implemented in internals_linux.go and a no-op in internals_windows.go. I could of course add a comment to explain why it is empty if that would be clearer (IMO that probably would be).